.github/workflows/CI.yml

```diff
@@ -133,4 +133,5 @@ jobs:
           dotnet new tool-manifest
           dotnet tool install fantomless-tool --version 4.7.997-prerelease
           dotnet fantomless --recurse .
+          git restore src/FileConventions.Test/DummyFiles
```

Comment:
@realmarv the test files for EolAtEof don't need to have .fsx extension; this way we don't need this `git restore` call at all